examples/client/README.md

```diff
@@ -55,9 +55,9 @@ With `send-data` we can publish new values on a server-owned interface of our lo
 instance.
 The syntax is the following:
 ```
-astartectl appengine --appengine-url http://localhost:4002/                       \
-    --realm-management-url http://localhost:4000/ --realm-key <REALM>_private.pem \
-    --realm-name <REALM> devices send-data <DEVICE_ID>                            \
+astartectl appengine --appengine-url http://api.astarte.localhost/appengine            \
+    --realm-management-url http://api.astarte.localhost/realmmanagement                \
+    --realm-key <REALM>_private.pem --realm-name <REALM> devices send-data <DEVICE_ID> \
```

Comment:
These changes differ from the other API URLs used in this Readme, so put back the previously configured URL.

Reply:
While rebasing I forgot to change also the other APIs, but for tests I used the new ones.

Reply:
Update also this [README.md](https://github.com/astarte-platform/astarte-message-hub/blob/master/examples/README.md)